script/start_services.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# 현재 스크립트가 위치한 디렉토리로 이동합니다.
+BASE_DIR=$(dirname "$0")
+cd "$BASE_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)